src/core/json/include/sourcemeta/core/json_array.h

```diff
@@ -56,31 +56,39 @@ template <typename Value> class JSONArray {
   /// Get a mutable end iterator on the array
   auto end() noexcept -> iterator { return this->data.end(); }
   /// Get a constant begin iterator on the array
-  auto begin() const noexcept -> const_iterator { return this->data.begin(); }
+  [[nodiscard]] auto begin() const noexcept -> const_iterator {
```

Comment:
I'm not sure this is working for me. I removed the `[[nodiscard]]` here, ran `make`, and it all compiles just fine?